abuild.in

```diff
@@ -2600,6 +2607,25 @@ if [ $(id -u) -eq 0 ] && [ -z "$FAKEROOTKEY" ]; then
 	FAKEROOT=
 fi
 
+# check for ccache presence and maybe export its path
+if [ -n "$USE_CCACHE" ] && [ ! -x "/usr/bin/ccache" ]; then
+	die "Ccache is enabled; however, the binary can not be found."
+elif [ -n "$USE_CCACHE" ]; then
+	if [ ! -d "$HOME/.ccache" ]; then
+		die "Ccache is enabled; however, the user cache directory is not found."
+	fi
+
+	if [ ! -f "$HOME/.ccache/ccache.conf" ]; then
+		die "Ccache is enabled; however, the ccache configuration file is not found."
+	fi
+
+	# prepend ccache path
+	export PATH="/usr/lib/ccache/bin:$PATH"
+
+	msg "ccache statistics"
+	/usr/bin/ccache -s
```

Comment:
do we really need to print ccache stats?

Reply:
Not really... :)